<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgDiocesisDetalles</class>
 <widget class="QWidget" name="dlgDiocesisDetalles">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>462</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detalles de diócesis</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tomo</string>
       </property>
       <property name="buddy">
        <cstring>spTomo</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spTomo">
       <property name="minimum">
        <number>1</number>
       </property>
       <property name="maximum">
        <number>8</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Año</string>
       </property>
       <property name="buddy">
        <cstring>spAnoRef</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spAnoRef">
       <property name="minimum">
        <number>1000</number>
       </property>
       <property name="maximum">
        <number>2000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_12">
       <property name="text">
        <string>Número de obispos</string>
       </property>
       <property name="buddy">
        <cstring>spNumeroObispos</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spNumeroObispos"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ckSinDatos">
       <property name="text">
        <string>Sin datos</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Rentas</string>
       </property>
       <property name="buddy">
        <cstring>spRenta</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spRenta">
       <property name="maximum">
        <number>1000000</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_10">
       <property name="text">
        <string>Divisa</string>
       </property>
       <property name="buddy">
        <cstring>txtRentasDivisa</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtRentasDivisa"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tasa</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="spTasa">
       <property name="maximum">
        <double>100000.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_11">
       <property name="text">
        <string>Divisa</string>
       </property>
       <property name="buddy">
        <cstring>txtTasaDivisa</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtTasaDivisa"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QLabel" name="label_9">
       <property name="text">
        <string>Sufragánea</string>
       </property>
       <property name="buddy">
        <cstring>txtSufraganea</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="myQLineEdit" name="txtSufraganea"/>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QCheckBox" name="ckInmediata">
       <property name="text">
        <string>Inmediata</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Señor temporal</string>
       </property>
       <property name="buddy">
        <cstring>txtSenorTemporal</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtSenorTemporal"/>
     </item>
     <item>
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Dignidades</string>
       </property>
       <property name="buddy">
        <cstring>spDignidades</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spDignidades"/>
     </item>
     <item>
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Canonicos</string>
       </property>
       <property name="buddy">
        <cstring>spCanonicos</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spCanonicos"/>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>Beneficiados</string>
       </property>
       <property name="buddy">
        <cstring>spBeneficiados</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spBeneficiados"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Libre</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="WidgetExtraInfos" name="wdExtras" native="true"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Notas</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="WidgetNotas" name="wdNotas" native="true"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btCancelar">
       <property name="text">
        <string>Can&amp;celar</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btOK">
       <property name="text">
        <string>&amp;Aceptar</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetExtraInfos</class>
   <extends>QWidget</extends>
   <header>src/widgets/widgetextrainfos.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetNotas</class>
   <extends>QWidget</extends>
   <header>src/widgets/widgetnotas.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>myQLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">src/widgets/myqlineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>spTomo</tabstop>
  <tabstop>spAnoRef</tabstop>
  <tabstop>spNumeroObispos</tabstop>
  <tabstop>ckSinDatos</tabstop>
  <tabstop>spRenta</tabstop>
  <tabstop>txtRentasDivisa</tabstop>
  <tabstop>spTasa</tabstop>
  <tabstop>txtTasaDivisa</tabstop>
  <tabstop>txtSufraganea</tabstop>
  <tabstop>ckInmediata</tabstop>
  <tabstop>txtSenorTemporal</tabstop>
  <tabstop>spDignidades</tabstop>
  <tabstop>spCanonicos</tabstop>
  <tabstop>spBeneficiados</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>btCancelar</tabstop>
  <tabstop>btOK</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
